### ISSUE TITLE
Cannot find libLLVM

### ISSUE THREAD

**Issue by DhairyaLGandhi** [Optomatica/heroku-buildpack-julia#5]
This seems to come from needing the .so via some dependency of Flux, which should be well supported, I would think

Stacktrace:
```
020-06-19T06:56:42.137294+00:00 app[web.1]: ERROR: LoadError: InitError: could not load library "/tmp/build_a645579a9258346bd758000f564b754e/julia/bin/../lib/julia/libLLVM-8jl.so"
2020-06-19T06:56:42.137362+00:00 app[web.1]: /tmp/build_a645579a9258346bd758000f564b754e/julia/bin/../lib/julia/libLLVM-8jl.so: cannot open shared object file: No such file or directory
2020-06-19T06:56:42.504387+00:00 app[web.1]: Stacktrace:
2020-06-19T06:56:42.624795+00:00 app[web.1]: [1] macro expansion at /app/.julia/packages/LLVM/pINgj/src/base.jl:18 [inlined]
2020-06-19T06:56:42.624910+00:00 app[web.1]: [2] LLVMResetFatalErrorHandler at /app/.julia/packages/LLVM/pINgj/lib/8.0/libLLVM_h.jl:109 [inlined]
2020-06-19T06:56:42.625044+00:00 app[web.1]: [3] _install_handlers at /app/.julia/packages/LLVM/pINgj/src/core/context.jl:112 [inlined]
2020-06-19T06:56:42.703688+00:00 app[web.1]: [4] __init__() at /app/.julia/packages/LLVM/pINgj/src/LLVM.jl:148
2020-06-19T06:56:43.034562+00:00 app[web.1]: [5] _include_from_serialized(::String, ::Array{Any,1}) at ./loading.jl:697
2020-06-19T06:56:43.035169+00:00 app[web.1]: [6] _require_search_from_serialized(::Base.PkgId, ::String) at ./loading.jl:781
2020-06-19T06:56:43.036013+00:00 app[web.1]: [7] _tryrequire_froBSON = "fbb218c0-5317-5bc6-957e-2ee96dd4b1f0"
m_serialized(::Base.PkgId, ::UInt64, ::String) at ./loading.jl:712
2020-06-19T06:56:43.037101+00:00 app[web.1]: [8] _require_search_from_serialized(::Base.PkgId, ::String) at ./loading.jl:770
2020-06-19T06:56:43.037310+00:00 app[web.1]: [9] _tryrequire_from_serialized(::Base.PkgId, ::UInt64, ::String) at ./loading.jl:712
2020-06-19T06:56:43.037527+00:00 app[web.1]: [10] _require_search_from_serialized(::Base.PkgId, ::String) at ./loading.jl:770
2020-06-19T06:56:43.037665+00:00 app[web.1]: [11] _require(::Base.PkgId) at ./loading.jl:1006
2020-06-19T06:56:43.037787+00:00 app[web.1]: [12] require(::Base.PkgId) at ./loading.jl:927
2020-06-19T06:56:43.038288+00:00 app[web.1]: [13] require(::Module, ::Symbol) at ./loading.jl:922
2020-06-19T06:56:43.038454+00:00 app[web.1]: [14] include(::Module, ::String) at ./Base.jl:377
2020-06-19T06:56:43.038697+00:00 app[web.1]: [15] exec_options(::Base.JLOptions) at ./client.jl:288
2020-06-19T06:56:43.038780+00:00 app[web.1]: [16] _start() at ./client.jl:484
2020-06-19T06:56:43.038801+00:00 app[web.1]: during initialization of module LLVM
2020-06-19T06:56:43.038801+00:00 app[web.1]: in expression starting at /app/utrain.jl:29
2020-06-19T06:56:43.101580+00:00 heroku[web.1]: Process exited with status 1
2020-06-19T06:56:43.138851+00:00 heroku[web.1]: State changed from starting to crashed
```

**Comment by DhairyaLGandhi** [comment]
I should note that the build goes through fine, as does the precompilation, but this happens while deploying

**Comment by amgad-naiem** [comment]
This have been fixed in the latest LLVM release v1.6.0. However Flux uses a lot of memory so you will have to use at least standard-2x dyno and [increase boot timeout](https://tools.heroku.support/limits/boot_timeout)
